src/deployment/web.md

```diff
@@ -109,6 +76,100 @@ and open the /build/web directory. Navigate to
 (given the python SimpleHTTPServer example)
 to view the release version of your app.
 
+## Deploying to the web
+
+When you are ready to deploy your app,
+upload the release bundle
+to Firebase, the cloud, or a similar service.
+Here are a few possibilities, but there are
+many others:
+
+* [Firebase Hosting][]
+* [GitHub Pages][]
+* [Google Cloud Hosting][]
+
+## Deploying to Firebase Hosting
+You can use the Firebase CLI to build and release your Flutter app with Firebase
+Hosting.
+
+### Before you begin
+To get started, [install or update][install-firebase-cli] the Firebase CLI:
+
+```
+npm install -g firebase-tools
+```
+
+### Initialize Firebase
+
+1. Enable the web frameworks preview to the [Firebase framework-aware CLI][]:
+
+    ```
+    firebase experiments:enable webframeworks
+    ```
+
+2. In an empty directory or an existing Flutter project, run the initialization
+command:
+
+    ```
+    firebase init hosting
+    ```
+
+3. Answer yes when asked if you want to use a web framework.
+
+4. If you are in an empty directory, you will be asked to choose your web
+framework. Choose Flutter Web.
+
+5. Choose your hosting source directory, this could be an existing flutter app.
+
+6. Select a region to host your files.
+
+7. Choose whether to set up automatic builds and deploys with GitHub.
+
+8. Deploy the app to Firebase Hosting:
+
+    ```
+    firebase deploy
+    ```
+
+    Running this command automatically runs `flutter build web --release` for you,
+    so you don't have to build your app in a separate step.
```

Comment:
**issue:** 
Write as shorter sentences per [GSG](https://developers.google.com/style/accessibility#ease-of-reading).
```suggestion
    By running this command, Flutter invokes `flutter build web --release`.
    You don't have to build your app in a separate step.
```